entities/project.ts

```diff
@@ -251,17 +251,23 @@ class Project extends BaseEntity {
         'user',
         'user.id = CAST(project.admin AS INTEGER)',
       )
-      .innerJoinAndSelect('project.categories', 'c')
-      .where(
-        `project.statusId = ${ProjStatus.active} AND project.listed = true`,
-      );
+      // .innerJoinAndSelect('project.categories', 'c')
+      // .where(
+      //   `project.statusId = ${ProjStatus.active} AND project.listed = true`,
+      // );
```

Comment:
@CarlosQ96 
Why did you comment these lines?
is it related to this task?

Reply:
whoops.

Reply:
Fixed.

Reply:
Thanks